src/azure-cli/azure/cli/command_modules/backup/custom_help.py

```diff
@@ -288,7 +297,7 @@ def get_or_read_json(json_or_file):
         with open(json_or_file) as f:
             json_obj = json.load(f)
     if json_obj is None:
-        raise ValueError(
+        raise ResourceNotFoundError(
```

Comment:
Maybe ValidationError here. ResourceNotFoundError refers to 404 error for azure resources